maskit/examples/load_mnist.py

```diff
@@ -0,0 +1,122 @@
+import tensorflow as tf
+import collections
+from sklearn.decomposition import PCA
+from pennylane import numpy as np
+from sklearn.preprocessing import minmax_scale
+
+np.random.seed(42)
+
+MAX_TRAIN_SAMPLES = 11471
+MAX_TEST_SAMPLES = 1952
+
+
+def reduce_image(x):
+    x = np.reshape(x, [1, 28, 28, 1])
```

Comment:
I'm a bit stumped here. Do these magic numbers have some meaning? I'm guessing the 28x28 is the mnist pixel size – what's the outer 1x1 for?

Reply:
The tensorflow resize function requires the structure [batch, height, width, channels]. So technically I could omit the first 1, because it's processing one image at a time.